README.md

```diff
@@ -157,6 +157,12 @@ The advanced properties that can be configured for the connector are:
 | client.use.binary.protocol | Set to `false` to force the library to use JSON encoding for REST and realtime operations, instead of msgpack encoding. | *Boolean* | True |
 | client.loglevel | Sets the verbosity of logging. | *Integer* | 0 |
 
+## Dynamic Channel Configuration
+You can configure your channels dynamically by using `#{topic}` and/or `#{key}` placeholders in some configuration values. If you specify `#{key}` in your value, you must send a UTF-8 encoded string as your key.
+Configurations that are supported:
+* `channel`
+  * For example. if you define a channel value with `channel_#{topic}_#{key}` in your configuration, and publish a message to "topic1" with key "key1", the channel will be configured with `channel_topic1_key1` value.
```

Comment:
Give there is no just one configuration option listed, perhaps this could be re-worded to avoid the list? For example:

"The `channel` configuration option can include a `#{topic}` and/or `#{key}` placeholder..."

Reply:
I thought about this but here https://github.com/ably/kafka-connect-ably/pull/48 we also add message name for configuration. If we have a different section for the later (something like "dynamic message name configuration"), then we could get rid of the list. Otherwise we would have to keep the list and maybe change the title a little bit. WDYT?

Reply:
"The `channel` and `message.name` configuration options can include a #{topic} and/or #{key} placeholder..."

🙂

Reply:
I'll leave it to your judgement